bootstrap/bukkit/src/main/java/org/geysermc/platform/bukkit/GeyserBukkitConfiguration.java

```diff
@@ -61,6 +66,11 @@ public GeyserBukkitConfiguration(File dataFolder, FileConfiguration config) {
         }
     }
 
+    public void loadFloodgate(GeyserBukkitPlugin plugin) {
+        Plugin floodgate = Bukkit.getPluginManager().getPlugin("floodgate-bukkit");
```

Comment:
Pass the plugin in the constructor instead of retrieving it from the plugin manager directly.